map/framework.hpp

```diff
@@ -75,7 +72,8 @@
 #include "std/target_os.hpp"
 #include "std/unique_ptr.hpp"
 #include "std/vector.hpp"
-#include "std/weak_ptr.hpp"
+
+#include "boost/optional.hpp"
```

Comment:
we had an agreement to use system boost, didn't we?

Reply:
Right, system boost.

Reply:
so shouldn't it be ```<boost/optional.hpp>```?

Reply:
Np